examples/informers/DeltaChangesQueryingController.cs

```diff
@@ -0,0 +1,111 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Threading.Tasks.Dataflow;
+using k8s;
+using k8s.Informers;
+using k8s.Informers.Notifications;
+using k8s.Models;
+using KellermanSoftware.CompareNetObjects;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace informers
+{
+    // this sample demos both informer and controller
+    // there are two loggers:
+    //   _informerLogger lets you see raw data coming out of informer stream
+    //   _reconcilerLogger lets you see batches of object transitions that object went through since last time we did work on it
+    // reconciler is purposely slowed down to show accumulation of events between worker actions
+
+    // try creating and deleting some pods in "default" namespace and watch the output
+
+    public class DeltaChangesQueryingController : IController
```

Comment:
This example has a lot of commented out code, I wonder if it might be better to just have two different files for the two different examples